rmbox/Makefile.am

```diff
@@ -1,12 +1,12 @@
-bin_PROGRAMS = rmbox logger
+bin_PROGRAMS = rmbox sof_logger
```

Comment:
convention is to use - instead of _ for naming. So it would be "sof-logger". rmbox should follow same convention too ("sof-mbox").

Reply:
@lgirdwood I've updated this.